packages/web-react/package.json

```diff
@@ -17,6 +17,7 @@
   "module": "./index.js",
   "types": "./index.d.ts",
   "dependencies": {
+    "@react-hook/resize-observer": "^1.2.6",
```

Comment:
You won't be suprised I'm not fond of having another dependency, but having observed your discussions last week, I'm sure there is a good reason to have it 🙂.

Reply:
We have also discussed it with @tomassychra. In this case, having a dependency is way better than adopting the implementation and testing of Resize Observer.